Assets/Scripts/RestaurantRush/RRGameMananger.cs

```diff
@@ -54,6 +55,12 @@ void Awake()
     }
     void Start()
     {
+        //Disable buttons
+        for (int i = 0; i < 4; i++)
```

Comment:
consider using the .count or size of list instead of hardcoded 4 for size. Or make a const int for the size. I might also consider just using an array instead of a list here as Lists have a bit more overhead and it looks like you only use it like an array.